src/main/java/decodex/modules/hex/HexDecoder.java

```diff
@@ -0,0 +1,38 @@
+package decodex.modules.hex;
+
+import decodex.data.Data;
+import decodex.modules.BaseModule;
+import java.util.Arrays;
+import java.util.stream.Collectors;
+
+public class HexDecoder extends BaseModule {
+
+    public static final String MODULE_NAME = "hex-decode";
+
+    public static final String MODULE_DESCRIPTION = "Converts a hexadecimal string back into its raw value";
+    public static final String MODULE_EXAMPLE = "\"74 61 6b 6f 79 61 6b 69\" becomes \"takoyaki\"";
+
+    public HexDecoder() {
+        super(MODULE_NAME, MODULE_DESCRIPTION);
+
```

Comment:
I think the whitespace here for line 17 of constructor isn't necessary since it doesn't affect readability here?

Reply:
Perhaps remove the newline at line 21?